src/sandstorm/sandstorm-http-bridge.c++

```diff
@@ -1238,7 +1290,19 @@ private:
     auto additionalHeaderList = context.getAdditionalHeaders();
     if (additionalHeaderList.size() > 0) {
       for (auto header : additionalHeaderList) {
-        lines.add(kj::str(header.getName(), ": ", header.getValue()));
+        if (checkIsHttpToken(header.getName()) && checkIsHeaderValue(header.getValue())) {
```

Comment:
Cap'n Proto string getters are not particularly efficient since they need to validate the pointer. Can you save getName() and getValue() into local vars first to avoid calling these repeatedly below?